src/codeu/chat/UserMap.java

```diff
@@ -0,0 +1,29 @@
+package codeu.chat;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Iterator;
+
+import codeu.chat.client.simplegui.NewAccount;
+
+/**
+ * Created by Suveena on 5/24/17.
+ */
+
+/* Basic structure that holds all the users. Create this map upon starting GUI.
+Each time we add a new user, check to make sure that they don't already exist.
+If they do not, add them to the map.*/
```

Comment:
These comments doesn't make sense to me. So you're keeping a "database" on the client side? If you have 3 users (each with their own GUI, but connected to each other), and each of them have their own hashmap, how are you going to keep the maps in sync?